Core/TimberApi/Versions.cs

```diff
@@ -7,7 +7,7 @@ public class Versions
     {
         public static readonly Version GameVersion = Version.Parse(Timberborn.Versioning.Versions.VersionNumber);
 
-        public static Version TimberApiVersion { get; } = Version.Parse("0.5.2.0".ReplacePlaceholderWithFakeVersion());
+        public static Version TimberApiVersion { get; } = Version.Parse("0.5.3.0".ReplacePlaceholderWithFakeVersion());
 
         internal static Version TimberApiMinimumGameVersion { get; } = new(0, 2, 8, 1);
```

Comment:
Probably minimunGameVersion needs to be upped to 0.3.0 too

Reply:
Oh yeah right. Maybe we need some github structure like main -> dev -> any other branch.
Than we can just push everything to dev and whenerver we want to release it goes to main and fix all those things.